### ISSUE TITLE
array|string deprecation messages

### ISSUE THREAD

**Issue by TheCartpenter** [opencart/opencart#11815]
@danielkerr: It seem this issue was reproduced about the array|string deprecation messages in the oc-3 repository: https://github.com/opencart/opencart-3/issues/118#issue-1411600044 .

**Comment by TheCartpenter** [comment]
Followed are all the list of files involving array|string:

- system/library/cache/apc.php
- system/library/cache/redis.php
- system/library/cache/file.php
- system/library/cache/mem.php
- system/library/cache/memcached.php
- system/library/cache.php

**Comment by danielkerr** [comment]
array|string deprecation messages

**Comment by TheCartpenter** [comment]
Yes, correct. We might need alternatives in order for users to be able to complete their checkout process since these messages looks like they appear based on the PHP 8.x series users may have installed.
